LRRServer/lawreadingrobot/models.py

```diff
@@ -0,0 +1,48 @@
+
+import uuid
+
+from sqlalchemy import JSON, Column, DateTime, Text, Uuid
+from sqlalchemy.sql import func
+
+from .database import Base
+
+
+class BaseMixin(object):
+    created_at = Column(DateTime(timezone=True), server_default=func.now())
+    updated_at = Column(DateTime(timezone=True), onupdate=func.now())
+    # Makes sure the columns are added to the end of the table
+    created_at._creation_order = 9998  # type: ignore
+    updated_at._creation_order = 9999  # type: ignore
+
+
+def new_uuid() -> uuid.UUID:
+    return uuid.uuid4()
+
+
+class HouseBill(Base, BaseMixin):
+    __tablename__: str = 'housebill'
+
+    id = Column(Uuid(as_uuid=True),
+                primary_key=True,
+                index=True,
+                unique=True,
+                default=new_uuid
+                )
+    hb_id = Column(Text, index=True)
+    title = Column(Text, index=True)
+    title_detail = Column(JSON)
```

Comment:
Consider removing the "*_details" fields. I think this is metadata for interpreting the "title" column, but this should probably be handled by whatever ingests the data, and shouldn't be stored here.